### PR TITLE
Con 2580 show banner continued apprenticeship

### PR DESCRIPTION
**Pull request by VasanthaKasirajan3008**

### CODE REVIEW

File: src/SFA.DAS.EAS.Web/Views/EmployerManageApprentices/Details.cshtml [hunk 1]
```diff
@@ -29,13 +29,84 @@
...
+                    <h3 class="heading-small">  View apprentice record </h3>
+                    <p>
+                        The apprentice moved providers during their apprenticeship.
+                    </p>
+                }
+                else if (Model.Data.HasApprovedChangeOfPartyRequest)
+                {
+                    <h3 class="heading-small">  View apprentice record </h3>
+                    <p>
+                        The apprentice moved providers during their apprenticeship.
```
Comment (hkf-tech) [inline src/SFA.DAS.EAS.Web/Views/EmployerManageApprentices/Details.cshtml:82]: the text accroding to the content deck : The apprentice moved providers during their apprenticeship.

File: src/SFA.DAS.EAS.Web/Views/EmployerManageApprentices/Details.cshtml [hunk 2]
```diff
@@ -29,13 +29,84 @@
...
+        }
+
+
+        @if (Model.Data.HasApprovedChangeOfPartyRequest || Model.Data.IsContinuation)
+        {
+
+            <div class="panel panel-border-wide alert-default">
+
+
+                @if (Model.Data.HasApprovedChangeOfPartyRequest && !Model.Data.IsContinuation)
```
Comment (hkf-tech) [inline src/SFA.DAS.EAS.Web/Views/EmployerManageApprentices/Details.cshtml:71]: It should be Model.Data.IsContinuation to cover the scenario where this record is middle record i.e. it has previous and next apprenticeship record.

**Review by hkf-tech — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.EAS.Web/Views/EmployerManageApprentices/Details.cshtml (hunk 2 above)
Comment (VasanthaKasirajan3008) [reply src/SFA.DAS.EAS.Web/Views/EmployerManageApprentices/Details.cshtml]: done

File: src/SFA.DAS.EAS.Web/Views/EmployerManageApprentices/Details.cshtml (hunk 1 above)
Comment (VasanthaKasirajan3008) [reply src/SFA.DAS.EAS.Web/Views/EmployerManageApprentices/Details.cshtml:82]: wordings are same, hence no change

**Review by hkf-tech — APPROVED** [review]

**Pull request merged**